### PR TITLE
build: exclude **/*.ts when publishing

### LINKED ISSUE #1403: Typescript error in icons-ui custom-tag component
### Expected Behaviour
I can import and use sp-popover without a typescript compile error.

### Actual Behaviour
I get a typescript compile error. I've tried excluding the problem file in my tsconfig but because I am importing the files they are still seen by tsc due to [module resolution](https://github.com/Microsoft/TypeScript/wiki/FAQ#why-is-a-file-in-the-exclude-list-still-picked-up-by-the-compiler) i guess.

#### Steps to Reproduce
Just importing the sp-popover and running tsc gets me the error. 
The workaround I ended up having to do so I could still use these components is not import the components
as es modules demonstrated in the sample code but to use script tags so i can bypass the typescript compilers module resolution.
```html
    <script src="./node_modules/@spectrum-web-components/popover/sp-popover.js" type="module"></script>
    <script src="./node_modules/@spectrum-web-components/overlay/overlay-trigger.js" type="module"></script>
```

#### Platform and Version
"lit-element": "^2.4.0",
"typescript": "^4.2.3",
"@spectrum-web-components/overlay": "^0.11.4",
"@spectrum-web-components/popover": "^0.9.6",

#### Sample Code that illustrates the problem
```ts
import '@spectrum-web-components/popover/sp-popover.js';
import '@spectrum-web-components/overlay/overlay-trigger.js';
```

```bash
npm run tsc
```

#### Logs taken while reproducing problem
```bash
node_modules/@spectrum-web-components/icons-ui/src/custom-tag.ts:30:5 - error TS2322: Type 'unknown' is not assignable to type 'string | TemplateResult'.
  Type 'unknown' is not assignable to type 'TemplateResult'.

 30     return values.reduce(
        ~~~~~~~~~~~~~~~~~~~~~
 31         (acc: string, v, idx) =>
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
...
 33         strings[0]
    ~~~~~~~~~~~~~~~~~~
 34     );
    ~~~~~~


Found 1 error.
```

### LINKED ISSUE #1532: Using bundle from jspm is not working
### Expected Behaviour
Fetching bundle from jspm should work.

### Actual Behaviour
It does not.

### Reproduce Scenario (including but not limited to)
Here is a link to a Lit playground I submitted in one of my previous issues here (#1509). It was working a few days ago, but not now. [Lit playground](https://lit.dev/playground/#project=W3sibmFtZSI6InNpbXBsZS1ncmVldGluZy50cyIsImNvbnRlbnQiOiJpbXBvcnQge2h0bWwsIGNzcywgTGl0RWxlbWVudH0gZnJvbSAnbGl0JztcbmltcG9ydCB7Y3VzdG9tRWxlbWVudCwgc3RhdGV9IGZyb20gJ2xpdC9kZWNvcmF0b3JzLmpzJztcblxuQGN1c3RvbUVsZW1lbnQoJ3NpbXBsZS1ncmVldGluZycpXG5leHBvcnQgY2xhc3MgU2ltcGxlR3JlZXRpbmcgZXh0ZW5kcyBMaXRFbGVtZW50IHtcbiAgQHN0YXRlKCkgcHJpdmF0ZSBpbnB1dENvdW50OiBudW1iZXIgPSAwO1xuICBAc3RhdGUoKSBwcml2YXRlIGNoYW5nZUNvdW50OiBudW1iZXIgPSAwO1xuICBcbiAgQHN0YXRlKCkgcHJpdmF0ZSBuYXRpdmVJbnB1dENvdW50OiBudW1iZXIgPSAwO1xuICBAc3RhdGUoKSBwcml2YXRlIG5hdGl2ZUNoYW5nZUNvdW50OiBudW1iZXIgPSAwO1xuICBcbiAgcmVuZGVyKCkge1xuICAgIHJldHVybiBodG1sYFxuICAgICAgPGgyIGNsYXNzPVwic3BlY3RydW0tSGVhZGluZyBzcGVjdHJ1bS1IZWFkaW5nLS1zaXplU1wiPlNwZWN0cnVtIE51bWJlciBGaWVsZDwvaDI-XG4gICAgICA8c3AtYmFubmVyPlxuICAgICAgICA8ZGl2IHNsb3Q9XCJoZWFkZXJcIj5JbnB1dCBjb3VudDwvZGl2PlxuICAgICAgICA8ZGl2IHNsb3Q9XCJjb250ZW50XCI-JHt0aGlzLmlucHV0Q291bnR9PC9kaXY-XG4gICAgICA8L3NwLWJhbm5lcj5cbiAgICAgIDxzcC1iYW5uZXI-XG4gICAgICAgIDxkaXYgc2xvdD1cImhlYWRlclwiPkNoYW5nZSBjb3VudDwvZGl2PlxuICAgICAgICA8ZGl2IHNsb3Q9XCJjb250ZW50XCI-JHt0aGlzLmNoYW5nZUNvdW50fTwvZGl2PlxuICAgICAgPC9zcC1iYW5uZXI-XG4gICAgICA8YnI-PGJyPlxuICAgICAgICBcbiAgICAgIDxzcC1udW1iZXItZmllbGRcbiAgICAgICAgQGlucHV0PVwiJHsoKSA9PiB0aGlzLmlucHV0Q291bnQrK31cIlxuICAgICAgICBAY2hhbmdlPVwiJHsoKSA9PiB0aGlzLmNoYW5nZUNvdW50Kyt9XCJcbiAgICAgICAgc3R5bGU9XCJ3aWR0aDogMjAwcHhcIlxuICAgICAgPjwvc3AtbnVtYmVyLWZpZWxkPlxuICAgICAgICBcbiAgICAgIDxicj48YnI-PGJyPlxuICAgICAgICBcbiAgICAgIDxoMiBjbGFzcz1cInNwZWN0cnVtLUhlYWRpbmcgc3BlY3RydW0tSGVhZGluZy0tc2l6ZVNcIj4mbHQ7aW5wdXRbdHlwZT1udW1iZXJdJmd0OzwvaDI-XG4gICAgICA8c3AtYmFubmVyPlxuICAgICAgICA8ZGl2IHNsb3Q9XCJoZWFkZXJcIj5OYXRpdmUgaW5wdXQgY291bnQ8L2Rpdj5cbiAgICAgICAgPGRpdiBzbG90PVwiY29udGVudFwiPiR7dGhpcy5uYXRpdmVJbnB1dENvdW50fTwvZGl2PlxuICAgICAgPC9zcC1iYW5uZXI-XG4gICAgICA8c3AtYmFubmVyPlxuICAgICAgICA8ZGl2IHNsb3Q9XCJoZWFkZXJcIj5OYXRpdmUgY2hhbmdlIGNvdW50PC9kaXY-XG4gICAgICAgIDxkaXYgc2xvdD1cImNvbnRlbnRcIj4ke3RoaXMubmF0aXZlQ2hhbmdlQ291bnR9PC9kaXY-XG4gICAgICA8L3NwLWJhbm5lcj5cbiAgICAgIDxicj48YnI-XG4gICAgICAgIFxuICAgICAgPGlucHV0XG4gICAgICAgIHR5cGU9XCJudW1iZXJcIlxuICAgICAgICBAaW5wdXQ9XCIkeygpID0-IHRoaXMubmF0aXZlSW5wdXRDb3VudCsrfVwiXG4gICAgICAgIEBjaGFuZ2U9XCIkeygpID0-IHRoaXMubmF0aXZlQ2hhbmdlQ291bnQrK31cIlxuICAgICAgICBzdHlsZT1cIndpZHRoOiAyMDBweFwiXG4gICAgICA-XG4gICAgYDtcbiAgfVxufVxuIn0seyJuYW1lIjoiaW5kZXguaHRtbCIsImNvbnRlbnQiOiI8IURPQ1RZUEUgaHRtbD5cbjxoZWFkPlxuICA8c2NyaXB0XG4gICAgc3JjPVwiaHR0cHM6Ly9qc3BtLmRldi9Ac3BlY3RydW0td2ViLWNvbXBvbmVudHMvYnVuZGxlL2VsZW1lbnRzLmpzXCJcbiAgICB0eXBlPVwibW9kdWxlXCJcbiAgPjwvc2NyaXB0PlxuICA8c2NyaXB0IHR5cGU9XCJtb2R1bGVcIiBzcmM9XCIuL3NpbXBsZS1ncmVldGluZy5qc1wiPjwvc2NyaXB0PlxuPC9oZWFkPlxuPGJvZHk-XG4gIDxzcC10aGVtZT5cbiAgICA8c2ltcGxlLWdyZWV0aW5nPjwvc2ltcGxlLWdyZWV0aW5nPlxuICA8L3NwLXRoZW1lPlxuPC9ib2R5PlxuIn1d)

The very first example in docs / Getting started is also not working:

image

#### Steps to Reproduce
Include following code:

```html
<script
    src="https://jspm.dev/@spectrum-web-components/bundle/elements.js"
    type="module"
></script>
```

#### Platform and Version
* Windows 10
* Chrome 91.0.4472.77

#### Logs taken while reproducing problem
```
GET https://jspm.dev/npm:@spectrum-web-components/thumbnail@0.2/sp-thumbnail.js net::ERR_ABORTED 404      elements.js:64
```

### PR DESCRIPTION
**Pull request by Westbrook**
## Description
Don't ship `*.ts` files to production to prevent errors (or worse the hacks) found in #1403.

I'll set up a pre-release of this soon to confirm our source maps, etc. still work as expected when consumed as a dependency in other projects. ✅ 

This will hopefully also address delivery on JSPM where the Thumbnail pattern chose not to work (while the immediately following `lit-next` pre-release did). As there is no structural basis for the failure, I can only assume it's something on JSPM's side and a new release will support it processing correctly this time 🤞🏼 

## Related Issue
fixes #1403
fixes #1532

## Motivation and Context
Easier consumption of our packages.
No CDN failures

## How Has This Been Tested?
Pre-release pending.

## Types of changes
- [x] Package inclusion changes

## Checklist:
- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by Westbrook** [comment]
Initial pre-release for this: 
```
Successfully published:
 - @spectrum-web-components/accordion@0.4.10-alpha.12+6493e508
 - @spectrum-web-components/action-bar@0.2.10-alpha.12+6493e508
 - @spectrum-web-components/action-button@0.5.2-alpha.12+6493e508
 - @spectrum-web-components/action-group@0.4.10-alpha.12+6493e508
 - @spectrum-web-components/action-menu@0.10.2-alpha.12+6493e508
 - @spectrum-web-components/asset@0.4.6-alpha.72+6493e508
 - @spectrum-web-components/avatar@0.7.6-alpha.72+6493e508
 - @spectrum-web-components/banner@0.6.4-alpha.146+6493e508
 - @spectrum-web-components/base@0.4.4-alpha.146+6493e508
 - @spectrum-web-components/bundle@0.22.1-alpha.0+6493e508
 - @spectrum-web-components/button-group@0.6.7-alpha.72+6493e508
 - @spectrum-web-components/button@0.14.2-alpha.12+6493e508
 - @spectrum-web-components/card@0.8.10-alpha.12+6493e508
 - @spectrum-web-components/checkbox@0.10.2-alpha.12+6493e508
 - @spectrum-web-components/coachmark@0.6.6-alpha.72+6493e508
 - @spectrum-web-components/color-area@0.1.6-alpha.12+6493e508
 - @spectrum-web-components/color-handle@0.1.5-alpha.72+6493e508
 - @spectrum-web-components/color-loupe@0.1.5-alpha.72+6493e508
 - @spectrum-web-components/color-slider@0.1.6-alpha.45+6493e508
 - @spectrum-web-components/color-wheel@0.1.6-alpha.45+6493e508
 - @spectrum-web-components/dialog@0.6.10-alpha.12+6493e508
 - @spectrum-web-components/divider@0.2.6-alpha.72+6493e508
 - @spectrum-web-components/dropzone@0.7.6-alpha.72+6493e508
 - @spectrum-web-components/field-group@0.3.6-alpha.72+6493e508
 - @spectrum-web-components/field-label@0.5.1-alpha.45+6493e508
 - @spectrum-web-components/icon@0.9.7-alpha.72+6493e508
 - @spectrum-web-components/icons-ui@0.6.7-alpha.72+6493e508
 - @spectrum-web-components/icons-workflow@0.6.7-alpha.72+6493e508
 - @spectrum-web-components/icons@0.7.6-alpha.146+6493e508
 - @spectrum-web-components/iconset@0.5.6-alpha.146+6493e508
 - @spectrum-web-components/illustrated-message@0.6.8-alpha.12+6493e508
 - @spectrum-web-components/link@0.9.2-alpha.12+6493e508
 - @spectrum-web-components/menu@0.8.8-alpha.12+6493e508
 - @spectrum-web-components/meter@0.4.2-alpha.12+6493e508
 - @spectrum-web-components/modal@0.3.6-alpha.72+6493e508
 - @spectrum-web-components/number-field@0.1.2-alpha.12+6493e508
 - @spectrum-web-components/overlay@0.11.8-alpha.12+6493e508
 - @spectrum-web-components/picker@0.5.2-alpha.12+6493e508
 - @spectrum-web-components/popover@0.9.10-alpha.12+6493e508
 - @spectrum-web-components/progress-bar@0.3.1-alpha.45+6493e508
 - @spectrum-web-components/progress-circle@0.2.6-alpha.72+6493e508
 - @spectrum-web-components/quick-actions@0.4.6-alpha.72+6493e508
 - @spectrum-web-components/radio@0.7.8-alpha.12+6493e508
 - @spectrum-web-components/search@0.8.10-alpha.12+6493e508
 - @spectrum-web-components/shared@0.12.5-alpha.45+6493e508
 - @spectrum-web-components/sidenav@0.10.3-alpha.12+6493e508
 - @spectrum-web-components/slider@0.10.1-alpha.0+6493e508
 - @spectrum-web-components/split-button@0.5.2-alpha.12+6493e508
 - @spectrum-web-components/split-view@0.2.7-alpha.12+6493e508
 - @spectrum-web-components/status-light@0.7.6-alpha.72+6493e508
 - @spectrum-web-components/styles@0.9.8-alpha.12+6493e508
 - @spectrum-web-components/switch@0.7.9-alpha.12+6493e508
 - @spectrum-web-components/tabs@0.6.3-alpha.12+6493e508
 - @spectrum-web-components/tags@0.6.10-alpha.12+6493e508
 - @spectrum-web-components/textfield@0.8.9-alpha.12+6493e508
 - @spectrum-web-components/theme@0.8.9-alpha.12+6493e508
 - @spectrum-web-components/thumbnail@0.2.1-alpha.12+6493e508
 - @spectrum-web-components/toast@0.8.10-alpha.12+6493e508
 - @spectrum-web-components/tooltip@0.8.10-alpha.12+6493e508
 - @spectrum-web-components/top-nav@0.3.3-alpha.12+6493e508
 - @spectrum-web-components/tray@0.0.3-alpha.45+6493e508
 - @spectrum-web-components/underlay@0.6.6-alpha.72+6493e508
 - @spectrum-web-components/story-decorator@0.3.11-alpha.0+6493e508
 - @spectrum-web-components/vrt-compare@0.0.8-alpha.12+6493e508
 ```

**Comment by Westbrook** [comment]
Looks like the source maps are still go to go here!

TS files loading from source map

**Comment by Westbrook** [comment]
This feels like it's in the right place now.

The `icons-*` packages are likely our most complex and they seem to be working out just fine: https://unpkg.com/browse/@spectrum-web-components/icons-workflow@0.6.7-alpha.74/

The styles package which is also fairly unique also looks good to go: https://unpkg.com/browse/@spectrum-web-components/styles@0.9.8-alpha.14/

And the perfect "normal" package, Button, also seems ready: https://unpkg.com/browse/@spectrum-web-components/button@0.14.2-alpha.14/

**Comment by Westbrook** [comment]
Also, see the report from our third party QA resources 😉 https://github.com/adobe/spectrum-web-components/issues/1403#issuecomment-860771820

Thanks @davidkudera 🙏🏼 for checking this out in your work to help confirm the fix!

**Review by najikahalsema — DISMISSED** [review]
Looks good to me! Maybe someone else can confirm?

**Review by stanvladut — APPROVED** [review]
Great work, looking good overall! 💯 
I only posted a small non-blocking comment.

File: tasks/build-packages.js [hunk 1]
```diff
@@ -11,18 +11,19 @@ OF ANY KIND, either express or implied. See the License for the specific languag
...
-    return exec(`yarn tsc --build ${paths}${watch ? ' -w' : ''}`);
+    return execSync(`yarn tsc --build ${paths} --verbose${watch ? ' -w' : ''}`);
 };
 
 export const buildPackages = async (watch) => {
     const paths = [];
     for await (const config of globby.stream(`./packages/*/tsconfig.json`)) {
         paths.push(config);
     }
+    console.log(paths.join(' '));
```
[the remark this answers is not in the document]
Comment (Westbrook) [reply tasks/build-packages.js:26]: Great call, was just adding https://github.com/adobe/spectrum-web-components/pull/1542 separately to help reduce the risk of this.

**Pull request merged**